src/test/java/com/exasol/adapter/document/files/connection/S3ConnectionPropertiesReaderTest.java

```diff
@@ -0,0 +1,47 @@
+package com.exasol.adapter.document.files.connection;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import org.junit.jupiter.api.Test;
+
+import com.exasol.adapter.document.connection.ConnectionPropertiesReader;
+
+class S3ConnectionPropertiesReaderTest {
+    @Test
+    void testReadOnlyRequired() {
+        final S3ConnectionProperties properties = runReader(
+                "{\"awsAccessKeyId\": \"myKey\", \"awsSecretAccessKey\": \"mySecretAccessKey\", "
+                        + "\"awsRegion\": \"eu-central-1\", \"s3Bucket\": \"my-bucket\" }");
+        assertAll(() -> assertThat(properties.getAwsAccessKeyId(), equalTo("myKey")),
+                () -> assertThat(properties.getAwsSecretAccessKey(), equalTo("mySecretAccessKey")),
+                () -> assertThat(properties.getAwsRegion(), equalTo("eu-central-1")),
+                () -> assertThat(properties.getS3Bucket(), equalTo("my-bucket")),
+                () -> assertThat(properties.getAwsEndpointOverride(), equalTo(null)),
+                () -> assertThat(properties.getAwsSessionToken(), equalTo(null)),
+                () -> assertThat(properties.isS3PathStyleAccess(), equalTo(false)),
+                () -> assertThat(properties.isUseSsl(), equalTo(true)));
+    }
+
```

Comment:
I assume it is already tested in common, but you can add here a test for missing one of required properties.